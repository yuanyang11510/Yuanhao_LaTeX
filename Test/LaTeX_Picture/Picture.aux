\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {1}图片原始大小}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}scale表示图片缩放倍数（0.3倍/0.5倍/0.7倍）}{1}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}width表示图片宽度（3cm/0.3倍页宽）}{1}{section.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {4}height表示图片高度（1cm/0.1倍页高）}{1}{section.4}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {5}angle表示图片逆时针旋转角度，origin表示图片旋转中心，默认值为l（左），可以设置为r（右）、c（中）、t（顶）、b（底）、B（基线）}{2}{section.5}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1}绕左侧中心逆时针旋转90度}{2}{subsection.5.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.2}绕左侧中心顺时针旋转90度}{2}{subsection.5.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.3}绕右侧中心逆时针旋转90度}{3}{subsection.5.3}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.4}绕中央中心逆时针旋转90度}{3}{subsection.5.4}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.5}绕中心顶端中心逆时针旋转90度}{4}{subsection.5.5}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.6}绕底部中心逆时针旋转90度}{4}{subsection.5.6}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.7}绕基线中心逆时针旋转90度}{5}{subsection.5.7}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {6}设置图片到边框的距离(原始尺寸/0.5cm)}{5}{section.6}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {7}在文本中插入图片}{6}{section.7}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1}不使用浮动体环境}{6}{subsection.7.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2}使用浮动体环境}{6}{subsection.7.2}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces doraemon\_figure}}{6}{figure.caption.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {8}图文混排}{6}{section.8}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces doraemon\_subfigure}}{7}{figure.caption.2}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {3}{\ignorespaces doraemon\_wrapfigure}}{7}{figure.caption.3}\protected@file@percent }
\gdef \@abspage@last{7}
