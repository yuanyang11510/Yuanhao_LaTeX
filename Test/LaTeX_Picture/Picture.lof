\contentsline {figure}{\numberline {1}{\ignorespaces doraemon\_figure}}{6}{figure.caption.1}%
\contentsline {figure}{\numberline {2}{\ignorespaces doraemon\_subfigure}}{7}{figure.caption.2}%
\contentsline {figure}{\numberline {3}{\ignorespaces doraemon\_wrapfigure}}{7}{figure.caption.3}%
