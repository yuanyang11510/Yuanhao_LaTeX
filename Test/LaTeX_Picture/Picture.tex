\documentclass{article}
\usepackage{ctex}
\usepackage{graphicx}
\usepackage{subcaption}
\usepackage{wrapfig}
\usepackage[colorlinks = true,linkcolor=blue]{hyperref}

%* 以下插入的图片外面都套有一层"\fbox{text}"命令，以清楚显示图片的边框界限

\begin{document}

\section{图片原始大小}
% graphicx宏包命令
    %picture in the same file as Tex
    \fbox{\includegraphics{doraemon1.jpg}}
    % XeLaTeX支持pdf、eps、png与jpg图片扩展名，所以可以写带扩展名的图片名称，也可以写不带扩展名的名称，如果不给出扩展名，将按上述4个扩展名的顺序依次搜索文件（吴康隆P50）

    % 绝对路径
    % \includegraphics{D:/OneDrive/（▲）Yuanhao_github/Yuanhao_LaTeX_github/Test/LaTeX_Picture/doraemon1.jpg} 
    %? 使用相对路径始终无法成功编译，不知道原因是什么

    %reserved file for pictures
    % \graphicspath{{D:/OneDrive/（▲）Yuanhao_github/Yuanhao_LaTeX_github/Test/LaTeX_Picture/Reserved file for pictures}}
    % \includegraphics{doraemon2.jpg}
    %* 注意，使用"\graphicspath{dir-list}"命令时，路径两边应该套用两层大括号，否则无法成功编译

\section{scale表示图片缩放倍数（0.3倍/0.5倍/0.7倍）}
    \fbox{\includegraphics[scale=.3]{doraemon1.jpg}}
    \fbox{\includegraphics[scale=.5]{doraemon1.jpg}}
    \fbox{\includegraphics[scale=.7]{doraemon1.jpg}}

\section{width表示图片宽度（3cm/0.3倍页宽）}

    \fbox{\includegraphics[width=3cm]{doraemon1.jpg}}
    \fbox{\includegraphics[width=.3\textwidth]{doraemon1.jpg}}

\section{height表示图片高度（1cm/0.1倍页高）}
    % 旋转的图片基线会变化，故一般用totalheight代替height（吴康隆P50）
    \fbox{\includegraphics[height=1cm]{doraemon1.jpg}}
    \fbox{\includegraphics[height=.1\textheight]{doraemon1.jpg}}

\section{angle表示图片逆时针旋转角度，origin表示图片旋转中心，默认值为l（左），可以设置为r（右）、c（中）、t（顶）、b（底）、B（基线）}
%* 顺时针旋转可以通过在旋转角度前面添加符号来完成
\subsection{绕左侧中心逆时针旋转90度}
    \fbox{\includegraphics[angle=90]{doraemon1.jpg}}%counterclockwise
    \fbox{\includegraphics{doraemon1.jpg}}

\subsection{绕左侧中心顺时针旋转90度}
    \fbox{\includegraphics[angle=-90]{doraemon1.jpg}}%clockwise
    \fbox{\includegraphics{doraemon1.jpg}}

\subsection{绕右侧中心逆时针旋转90度}
    \fbox{\includegraphics{doraemon1.jpg}}
    \fbox{\includegraphics[angle=90, origin=r]{doraemon1.jpg}}

\subsection{绕中央中心逆时针旋转90度}
    \fbox{\includegraphics{doraemon1.jpg}}
    \fbox{\includegraphics[angle=90, origin=c]{doraemon1.jpg}}

\subsection{绕中心顶端中心逆时针旋转90度}
    \fbox{\includegraphics{doraemon1.jpg}}
    \fbox{\includegraphics[angle=90, origin=t]{doraemon1.jpg}}

\subsection{绕底部中心逆时针旋转90度}
    \fbox{\includegraphics{doraemon1.jpg}}
    \fbox{\includegraphics[angle=90, origin=b]{doraemon1.jpg}}

\subsection{绕基线中心逆时针旋转90度}
    \fbox{\includegraphics{doraemon1.jpg}}
    \fbox{\includegraphics[angle=90, origin=B]{doraemon1.jpg}}

\section{设置图片到边框的距离(原始尺寸/0.5cm)}
    \fbox{\includegraphics{doraemon1.jpg}}
    \setlength{\fboxsep}{.5cm}
    \fbox{\includegraphics{doraemon1.jpg}}

\section{在文本中插入图片}
\subsection{不使用浮动体环境}
    This is a picture \fbox{\includegraphics{doraemon1.jpg}} inserted into the text.

\subsection{使用浮动体环境}
    This is a picture independent of the text.
    \begin{figure}[htbp]
        \centering
        \fbox{\includegraphics{doraemon1.jpg}}
        \caption{doraemon\_figure}
    \end{figure}

% subcaption宏包命令
    \begin{figure}[htbp]
        \centering
        \begin{subfigure}{.3\textwidth}%this argument of width is obligatory
            \centering
            \fbox{\includegraphics[scale=.1]{doraemon1.jpg}}
            \subcaption{doraemon (1)}
        \end{subfigure}
        %
        \begin{subfigure}{.3\textwidth}
            \centering
            \fbox{\includegraphics[scale=.3]{doraemon1.jpg}}
            \subcaption{doraemon (2)}
        \end{subfigure}
        %
        \begin{subfigure}{.3\textwidth}
            \centering
            \fbox{\includegraphics[scale=.5]{doraemon1.jpg}}
            \caption{doraemon (3)} % subfigure环境中既可以使用"\subcaption{}"命令，也可以使用"\caption{}"命令
        \end{subfigure}	
        \caption{doraemon\_subfigure}	
    \end{figure}

% \clearpage
%rfr 使用"\clearpage"命令来清空浮动队列，并且新开一页，如果此处不使用这一项命令，则"doraemon_subfigure"一图会作为浮动体被插入到下面的文本当中，关于"\newpage"命令和"\clearpage"命令的区别，可以参考：https://blog.csdn.net/weixin_45008608/article/details/115835064、https://www.latexstudio.net/archives/8005.html

\section{图文混排}
% wrapfig宏包命令
    Texte texte texte texte texte texte texte
    texte texte texte texte texte texte texte
    texte texte texte texte texte texte...
    Texte texte texte texte texte texte texte
    texte texte texte texte texte texte texte
    texte texte texte texte texte texte...

    \begin{wrapfigure}{l}{5cm}%Notice that this is the argument of width
        \centering
        \fbox{\includegraphics[scale=.3]{doraemon1.jpg}}
        \caption{doraemon\_wrapfigure}
    \end{wrapfigure}
    Texte texte texte texte texte texte texte
    texte texte texte texte texte texte texte
    texte texte texte texte texte texte...
    Texte texte texte texte texte texte texte
    texte texte texte texte texte texte texte
    texte texte texte texte texte texte...
    Texte texte texte texte texte texte texte
    texte texte texte texte texte texte texte
    texte texte texte texte texte texte...
    Texte texte texte texte texte texte texte
    texte texte texte texte texte texte texte
    texte texte texte texte texte texte...
    % wrapfigure环境有两个必选参数，两个可选参数：[linenum]{place}[overhang]{picwidth}，必选参数包括{place}和{picwidth}，{place}可以选择r、l、i、o，分别代表图片在文字段的左侧、右侧、近书脊、远书脊，{picwidth}代表图片的宽度，图片的高度会自动调整；可选参数包括[linenum]和[overhang]，[linenum]代表图片所占行数，一般不指定，[overhang]代表允许图片超出页面文本区的宽度，默认是0pt，即不允许图片超出页面文本区，该项可以使用"\width"代替图片的宽度，填入"\width"将允许把图片全部放入页边区域（吴康隆P51）

\listoffigures
%* 插图目录显示所有在浮动体环境中使用"\caption{heading}"命令对应的图片所在页码
\end{document}