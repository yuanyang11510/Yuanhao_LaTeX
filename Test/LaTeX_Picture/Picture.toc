\contentsline {section}{\numberline {1}图片参数}{2}{section.1}%
\contentsline {subsection}{\numberline {1.1}图片原始大小}{2}{subsection.1.1}%
\contentsline {subsection}{\numberline {1.2}scale表示图片缩放倍数（0.3倍/0.5倍/0.7倍）}{2}{subsection.1.2}%
\contentsline {subsection}{\numberline {1.3}width表示图片宽度（3cm/0.3倍文本宽度）}{2}{subsection.1.3}%
\contentsline {subsection}{\numberline {1.4}height表示图片高度（1cm/0.1倍页高）}{2}{subsection.1.4}%
\contentsline {subsection}{\numberline {1.5}angle表示图片逆时针旋转角度，origin表示图片旋转中心，默认值为l（左），可以设置为r（右）、c（中）、t（顶）、b（底）、B（基线）}{3}{subsection.1.5}%
\contentsline {subsubsection}{\numberline {1.5.1}绕左侧中心逆时针旋转90度}{3}{subsubsection.1.5.1}%
\contentsline {subsubsection}{\numberline {1.5.2}绕左侧中心顺时针旋转90度}{3}{subsubsection.1.5.2}%
\contentsline {subsubsection}{\numberline {1.5.3}绕右侧中心逆时针旋转90度}{4}{subsubsection.1.5.3}%
\contentsline {subsubsection}{\numberline {1.5.4}绕中央中心逆时针旋转90度}{4}{subsubsection.1.5.4}%
\contentsline {subsubsection}{\numberline {1.5.5}绕中心顶端中心逆时针旋转90度}{5}{subsubsection.1.5.5}%
\contentsline {subsubsection}{\numberline {1.5.6}绕底部中心逆时针旋转90度}{5}{subsubsection.1.5.6}%
\contentsline {subsubsection}{\numberline {1.5.7}绕基线中心逆时针旋转90度}{6}{subsubsection.1.5.7}%
\contentsline {subsection}{\numberline {1.6}设置图片到边框的距离(原始尺寸/1cm)}{6}{subsection.1.6}%
\contentsline {section}{\numberline {2}在文本中插入图片}{7}{section.2}%
\contentsline {subsection}{\numberline {2.1}不使用浮动体环境}{7}{subsection.2.1}%
\contentsline {subsection}{\numberline {2.2}使用浮动体环境}{7}{subsection.2.2}%
\contentsline {section}{\numberline {3}图文混排}{7}{section.3}%
\contentsline {section}{\numberline {4}并排插入图片}{8}{section.4}%
\contentsline {subsection}{\numberline {4.1}并排插入共享标题的图片}{8}{subsection.4.1}%
\contentsline {subsection}{\numberline {4.2}并排插入拥有小标题的图片}{9}{subsection.4.2}%
\contentsline {subsection}{\numberline {4.3}并排插入拥有子编号的图片}{11}{subsection.4.3}%
\contentsline {subsubsection}{\numberline {4.3.1}subfig宏包}{11}{subsubsection.4.3.1}%
\contentsline {subsubsection}{\numberline {4.3.2}subcaption宏包}{11}{subsubsection.4.3.2}%
\contentsline {subsection}{\numberline {4.4}并排插入不同比例的图片}{13}{subsection.4.4}%
\contentsline {subsection}{\numberline {4.5}结论}{16}{subsection.4.5}%
