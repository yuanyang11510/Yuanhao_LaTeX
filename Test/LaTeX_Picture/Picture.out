\BOOKMARK [1][-]{section.1}{\376\377\126\376\162\107\123\237\131\313\131\047\134\017}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000s\000c\000a\000l\000e\210\150\171\072\126\376\162\107\177\051\145\076\120\015\145\160\377\010\0000\000.\0003\120\015\000/\0000\000.\0005\120\015\000/\0000\000.\0007\120\015\377\011}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000w\000i\000d\000t\000h\210\150\171\072\126\376\162\107\133\275\136\246\377\010\0003\000c\000m\000/\0000\000.\0003\120\015\230\165\133\275\377\011}{}% 3
\BOOKMARK [1][-]{section.4}{\376\377\000h\000e\000i\000g\000h\000t\210\150\171\072\126\376\162\107\232\330\136\246\377\010\0001\000c\000m\000/\0000\000.\0001\120\015\230\165\232\330\377\011}{}% 4
\BOOKMARK [1][-]{section.5}{\376\377\000a\000n\000g\000l\000e\210\150\171\072\126\376\162\107\220\006\145\366\224\210\145\313\217\154\211\322\136\246\377\014\000o\000r\000i\000g\000i\000n\210\150\171\072\126\376\162\107\145\313\217\154\116\055\137\303\377\014\236\330\213\244\120\074\116\072\000l\377\010\135\346\377\011\377\014\123\357\116\345\213\276\177\156\116\072\000r\377\010\123\363\377\011\060\001\000c\377\010\116\055\377\011\060\001\000t\377\010\230\166\377\011\060\001\000b\377\010\136\225\377\011\060\001\000B\377\010\127\372\176\277\377\011}{}% 5
\BOOKMARK [2][-]{subsection.5.1}{\376\377\176\325\135\346\117\247\116\055\137\303\220\006\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 6
\BOOKMARK [2][-]{subsection.5.2}{\376\377\176\325\135\346\117\247\116\055\137\303\230\172\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 7
\BOOKMARK [2][-]{subsection.5.3}{\376\377\176\325\123\363\117\247\116\055\137\303\220\006\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 8
\BOOKMARK [2][-]{subsection.5.4}{\376\377\176\325\116\055\131\056\116\055\137\303\220\006\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 9
\BOOKMARK [2][-]{subsection.5.5}{\376\377\176\325\116\055\137\303\230\166\172\357\116\055\137\303\220\006\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 10
\BOOKMARK [2][-]{subsection.5.6}{\376\377\176\325\136\225\220\350\116\055\137\303\220\006\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 11
\BOOKMARK [2][-]{subsection.5.7}{\376\377\176\325\127\372\176\277\116\055\137\303\220\006\145\366\224\210\145\313\217\154\0009\0000\136\246}{section.5}% 12
\BOOKMARK [1][-]{section.6}{\376\377\213\276\177\156\126\376\162\107\122\060\217\271\150\106\166\204\215\335\171\273\000\050\123\237\131\313\134\072\133\370\000/\0000\000.\0005\000c\000m\000\051}{}% 13
\BOOKMARK [1][-]{section.7}{\376\377\127\050\145\207\147\054\116\055\143\322\121\145\126\376\162\107}{}% 14
\BOOKMARK [2][-]{subsection.7.1}{\376\377\116\015\117\177\165\050\155\156\122\250\117\123\163\257\130\203}{section.7}% 15
\BOOKMARK [2][-]{subsection.7.2}{\376\377\117\177\165\050\155\156\122\250\117\123\163\257\130\203}{section.7}% 16
\BOOKMARK [1][-]{section.8}{\376\377\126\376\145\207\155\367\143\222}{}% 17
